frontend/src/app/homepage-offer/offers.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Offer } from './offers.model';
-import { Injectable } from '@angular/core';
+import { Injectable, isDevMode } from '@angular/core';
```

Comment:
Instead of using `isDevMode` here I would suggest to use `environments` and store this information per environment that we maintain.

Reply:
All right, will do.